h2o-r/h2o-package/R/classes.R

```diff
@@ -179,6 +179,12 @@ setMethod("show", "H2OModel", function(object) {
   if( !is.null(model.parts$xm) ) print(model.parts$xm)
   cat("\n")
   if( !is.null(model.parts$xms) ) print(model.parts$xms)
+    
+  # model's parameters
+  has_model_params <- !is.null(m$model_parameters)
```

Comment:
Purpose of this code is to print parameters by default ? I don't think this we want to change this behavior for our users, we only want to expose the parameters.